tests/nlp/utils/test_properties.py

```diff
@@ -38,3 +37,20 @@ def test_calculate_default_properties():
 
     # Assert
     assert_that(result, equal_to({'Toxicity': [0] * 20_000}))
+
+
+def test_skipping_long_calculation_properties(tweet_emotion_train_test_textdata):
+    
+    # Arrange
+    _, test = tweet_emotion_train_test_textdata
+    test_text = test.text
+
+    # Act
+    result = calculate_default_properties(test_text, include_properties=['Lexical Density', 'Unique Noun Count'])[0]
+    result_none_text = calculate_default_properties([None], include_properties=['Lexical Density', 'Unique Noun Count'])[0]
+
+    # Assert
+    assert_that(result['Lexical Density'][0: 10], equal_to([94.44, 93.75, 100.0, 91.67, 87.5, 100.0, 100.0, 100.0, 91.67, 91.67]))
+    assert_that(result['Unique Noun Count'][0: 10], equal_to([9, 2, 3, 3, 4, 10, 4, 2, 7, 5]))
+    assert_that(result_none_text['Lexical Density'], equal_to([np.nan]))
+    assert_that(result_none_text['Unique Noun Count'], equal_to([np.nan]))
```

Comment:
you may need to add a line here for Code Style CI

Reply:
Done